openedx_learning/apps/authoring/publishing/admin.py

```diff
@@ -246,3 +250,261 @@ def get_queryset(self, request):
         queryset = super().get_queryset(request)
         return queryset.select_related("learning_package", "changed_by") \
                        .annotate(num_changes=Count("records"))
+
+
+def _entity_list_detail_link(el: EntityList) -> SafeText:
+    """
+    A link to the detail page for an EntityList which includes its PK and length.
+    """
+    return model_detail_link(el, f"EntityList #{el.pk} with {el.entitylistrow_set.count()} row(s)")
+
+
+class ContainerVersionInlineForContainer(admin.TabularInline):
+    """
+    Inline admin view of ContainerVersions in a given Container
+    """
+    model = ContainerVersion
+    ordering = ["-publishable_entity_version__version_num"]
+    fields = [
+        "uuid",
+        "version_num",
+        "title",
+        "children",
+        "created",
+        "created_by",
+    ]
+    readonly_fields = fields  # type: ignore[assignment]
+    extra = 0
+
+    def get_queryset(self, request):
+        return super().get_queryset(request).select_related(
+            "publishable_entity_version"
+        )
+
+    def children(self, obj: ContainerVersion):
+        return _entity_list_detail_link(obj.entity_list)
+
+
+@admin.register(Container)
+class ContainerAdmin(ReadOnlyModelAdmin):
+    """
+    Django admin configuration for Container
+    """
+    list_display = ("key", "uuid", "created", "draft", "published", "see_also")
+    fields = [
+        "publishable_entity",
+        "learning_package",
+        "draft",
+        "published",
+        "created",
+        "created_by",
+        "see_also",
+        "most_recent_parent_entity_list",
+    ]
+    readonly_fields = fields  # type: ignore[assignment]
+    search_fields = ["publishable_entity__uuid", "publishable_entity__key"]
+    inlines = [ContainerVersionInlineForContainer]
+
+    def uuid(self, obj: Container) -> SafeText:
+        return model_detail_link(obj, obj.uuid)
+
+    def learning_package(self, obj: Container) -> SafeText:
+        return model_detail_link(
+            obj.publishable_entity.learning_package,
+            obj.publishable_entity.learning_package.key,
+        )
+
+    def get_queryset(self, request):
+        return super().get_queryset(request).select_related(
+            "publishable_entity",
+            "publishable_entity__learning_package",
+            "publishable_entity__published__version",
+            "publishable_entity__draft__version",
+        )
+
+    def draft(self, obj: Container) -> SafeText:
+        """
+        Link to this Container's draft ContainerVersion
+        """
+        if draft := obj.versioning.draft:
+            return format_html(
+                "Version {} ({})", draft.version_num, _entity_list_detail_link(draft.entity_list)
```

Comment:
Nit/optional: I think the `uuid` is not very useful, and the title is, so I suggest taking out the former and adding the latter.

```suggestion
                'Version {} "{}" ({})', draft.version_num, draft.title, _entity_list_detail_link(draft.entity_list)
```

Screenshot:

<img width="1407" height="113" alt="Screenshot 2025-07-16 at 3 19 36 PM" src="https://github.com/user-attachments/assets/6a1db14b-96f8-43a0-84de-cb42ffc6b47e" />

vs.

<img width="1431" height="114" alt="Screenshot 2025-07-16 at 3 20 07 PM" src="https://github.com/user-attachments/assets/88f6dfb9-c308-4683-b8a1-d2ae61213f96" />

Reply:
Agreed. Changed for both Draft and Published.

I also removed all other occurrences of UUID from the various container-related admin lists, detail views, and inlines. We're not using the UUIDs for anything right now, so I'd rather it not clutter up the useful information with something irrelevant. If an operator really needs to know something's UUID, it is available on the linked PublishableEntity detail view.